src/libraries/pretest.proj

```diff
@@ -35,7 +47,7 @@
   <UsingTask TaskName="GenerateFileVersionProps" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
   <Target Name="GenerateFileVersionProps"
           DependsOnTargets="CollectSharedFrameworkRuntimeFiles"
-          Condition="'$(PlatformManifestFile)' != '' and '$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'"
+          Condition="'$(PlatformManifestFile)' != '' and '$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' and '$(BinplaceTestSharedFramework)' == 'true'"
```

Comment:
Why is this condition added? `BinplaceTestSharedFramework` is always set to true when `BuildTargetFramework == NetCoreAppCurrent`.

Reply:
Perhaps you missed to set that to false when targetting mobile? I asked that here: https://github.com/dotnet/runtime/pull/34662/files#r407798496

Reply:
Leftover from when I removed `<BinPlaceTestSharedFramework Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android'">false</BinPlaceTestSharedFramework>`